src/mutators.rs

```diff
@@ -127,6 +130,33 @@ pub async fn send_mutations(
         .await?;
 
     println!("Sent mutation: {}", res.status());
-
     Ok(())
 }
+
+#[cfg(test)]
+mod test {
+    use super::create_mutations;
+    use std::collections::HashMap;
+    use std::fs;
+
+    #[test]
+    fn test_mutators() {
+        let file = fs::read_to_string("mocks/mutators.json").expect("Unable to read file");
+        let data: HashMap<String, Vec<String>> = serde_json::from_str(&file).unwrap();
+
+        for (to_be_mutated, results) in data {
+            let mut mutations: Vec<(usize, String)> = vec![];
+            let lines: Vec<&str> = vec![&to_be_mutated];
+            let mut success = true;
+            create_mutations(&lines, &mut mutations);
+            assert_eq!(mutations.len(), results.len());
+            for mutation in mutations {
+                if !results.contains(&mutation.1) {
+                    success = false;
+                }
+            }
```

Comment:
I think it should iterate over the expected array and not the resulting array. I think that's the reason why the second mutation wasn't caught (the `!=` one).

Reply:
I think now it's caughting everything right. I can't see difference of iterating the expected array or the resulting array, they should contain same stuff, so since mutations is a tuple maybe the way I did it makes it simpler?